CITATION.rst

```diff
@@ -3,20 +3,18 @@ Acknowledging or Citing SunPy
 
 If you have used SunPy in your scientific work we would appreciate it if you would acknowledge it.
 The continued growth and development of SunPy is dependent on the community being aware of the use SunPy.
-If you use SunPy, we therefore ask that you acknowledge SunPy appropriately in a publication, presentation, poster, or talk.
 
--  **For a publication**, we recommend the following line be added to
-   the conclusion or acknowledgements:
+Publications
+------------
 
-   *This research has made use of SunPy, an open-source and free
+Please add the following line to the conclusion or acknowledgements:
+
+   *This research has made use of SunPy vX.Y, an open-source and free
    community-developed solar data analysis package written in Python
    (citation).*
 
-   The citation is to the `SunPy v0.5 paper`_/`arXiv(open access)`_. If
-   the journal allows please also include a link to sunpy.org. If you
-   have the time, please email us to let us know about your paper, as we
```

Comment:
@dpshelio I have dropped this in a quest for simplicity of instruction. After all ADS provides this right? I am happy to put it back if you have strong feelings about it.

Reply:
Yeah, I don't see the need to be emailed.